GliaWidgets/Feature.swift

```diff
@@ -10,7 +10,9 @@ public struct Features: OptionSet {
 
     /// Bubble that is shown in engagement time outside of engagement view.
     public static let bubbleView = Self(rawValue: 1 << 0)
+    #if DEBUG
+    #warning("Remove 'secureConversations' flag once implementation is complete.")
     public static let secureConversations = Self(rawValue: 2 << 0)
-
-    public static let all: Self = [.bubbleView]
+    #endif
+    public static let all: Self = [.bubbleView, .secureConversations]
```

Comment:
This will raise compilation error during release build since `.secureConversations` is not available under Release configuration.

```swift
    #if DEBUG
    #warning("Remove 'secureConversations' flag once implementation is complete.")
    public static let secureConversations = Self(rawValue: 2 << 0)
    public static let all: Self = [.bubbleView, .secureConversations]
    #else
    public static let all: Self = [.bubbleView]
    #endif
```

Reply:
Removed it e3895d9eeef8037c7e6859086da98ed32129106b.